BC-MAR-21-ROR/kata-07-rejoj-checador-equipo-03-k07
# Feature#3 add devise for admin

# Description

Add devise authentication for administrator:

 - Implement devise gem and customize to allow minimalistic admin authentication.
 - Implement bootstrap_form gem to simplify views form syntax and styles  for autogenerated  devise form, and in the future can facilitate us other forms. 
 - Create admin with db:seed to initialize database.
 - Create home controller to see index and can facilitate sign_out with a link

See issue #3  for more information.
 
 # Testing

- bundle install
- rails server
- rails db:setup
- navigate to: http://localhost:3000/
- sign in with email: admin@example.com and password: password  
- after a successful sign in, click in sign out  to destroy session. 
- can test with other password or user to see failed login feedback.
  
  ## Screenshots 

![Image 2021-04-14 at 2 15 18 p m](https://user-images.githubusercontent.com/11895273/114710641-4369dd80-9d2e-11eb-9ec3-5aaeac19623e.jpg)

![Image 2021-04-14 at 2 13 43 p m](https://user-images.githubusercontent.com/11895273/114710619-3947df00-9d2e-11eb-9829-de9cda3c75dd.jpg)
